source/Irrlicht/CMakeLists.txt

```diff
@@ -393,6 +384,43 @@ add_library(IrrlichtMt
 	$<TARGET_OBJECTS:IRRGUIOBJ>
 )
 
+# Alias target provides add_submodule compatibility
+add_library(IrrlichtMt::IrrlichtMt ALIAS IrrlichtMt)
+
+target_include_directories(IrrlichtMt
+	PUBLIC
+		"$<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include/>"
+		"$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>"
+		"$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/irrlichtmt>"
+	PRIVATE
+		"$<$<BOOL:OGL_ENABLED>:${OPENGL_INCLUDE_DIR}>"
+		"$<$<BOOL:OGLES2_ENABLED>:${OPENGLES2_INCLUDE_DIR}>"
+
+		"$<$<BOOL:ANDROID>:${ANDROID_NDK}/sources/android/native_app_glue}>"
+		"$<$<BOOL:REQUIRE_X11>:${X11_INCLUDE_DIR}>"
+)
+
+target_link_libraries(IrrlichtMt
+	PUBLIC
+		JPEG::JPEG
```

Comment:
This target does not exist in [CMake 3.5](https://github.com/Kitware/CMake/blob/v3.5.2/Modules/FindJPEG.cmake), so you can't rely on it.

Reply:
Thank you for the link, I will check targets against it to ensure the correct compatibility.